core/model/modx/processors/resource/trash/getlist.class.php

```diff
@@ -55,7 +55,7 @@ public function prepareQueryBeforeCount(xPDOQuery $c)
         if ($deleted = $this->getDeleted()) {
             $c->where(['modResource.id:IN' => $deleted]);
         } else {
-            return;
+            $c->where(['modResource.id' => 0]);
```

Comment:
This can be slightly improved in same way as it's handled in 3.x.

```
$c->where(['modResource.id' => 0]);
return $c;
```

There's no need to keep building other where conditions.